P/pocl/build_tarballs.jl

```diff
@@ -130,6 +134,70 @@ init_block = """
             ENV["SDKROOT"] = "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk"
         end
     end
+
+    # expose JLL binaries to the library
+    # XXX: Scratch.jl is unusably slow with JLLWrapper-emitted @compiler_options
```

Comment:
I'm a little surprised that's so slow; `@get_scratch!()` [should immediately call `Scratch.get_scratch()`](https://github.com/JuliaPackaging/Scratch.jl/blob/d68e430dee44a1b4995be06d675491cf5c080880/src/Scratch.jl#L301-L308), so I would expect that the compilation work is already done for `get_scratch()`.  Maybe it's inlining it, and that's what causes the blowup in compile/runtime?

Reply:
The culprit is `@compiler_options infer=false`. Most of the slowdown comes from `Scratch.track_scratch_access`; MWE:

```julia
module Foo

Base.Experimental.@compiler_options infer=false

using Scratch

__init__() = Scratch.track_scratch_access(Base.UUID("0a6951bc-db18-734b-110e-239e1a40b903"), "whatever")

end
```

`infer=false` incurs a 160ms penalty or so